sink/sink

```diff
@@ -195,22 +195,23 @@ class GitHub(object):
 
     def push(self, status):
         github = status.get("github", { })
+        http = status.get("github", { })
         token = github.get("token", self.token)
-        requests = github.get("requests", [])
+        requests = github.get("requests", []) + http.get("requests", [])
 
         for r in requests:
             self.results['link'] = status['link']
             result_name = r.get('result', None)
 
-            result = self.req(token, r)
+            result = self.req(r.get('token'), r)
```

Comment:
Is there an `or token` missing, or will the request now always have the token due to the changes in `req()` above? The `token =` assingment from above is still being used further down in `self.watches(token, watches)`, does that not need to use the request specific token(s) as well?

Reply:
Thanks. Good catch. Fixed.